packages/webapp/.env

```diff
@@ -17,3 +17,6 @@ NEXT_PUBLIC_APP_NAME = "Eden Community App"
 NEXT_PUBLIC_EDEN_CONTRACT_ACCOUNT = "test.edev"
 NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT="10.00000000 WAX"
 # NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT="10.0000 EOS"
+IPFS_PINATA_JWT="<set-pinata-jwt-here>"
+IPFS_PINATA_API="https://api.pinata.cloud/psa"
+IPFS_UPLOAD_ENDPOINT_URL="https://api.pinata.cloud/pinning/pinFileToIPFS"
```

Comment:
Are we using this anywhere? Looks like we're just using the Pinning Services API instead.

Reply:
Removed.